.github/workflows/tests.yml

```diff
@@ -77,6 +77,7 @@ jobs:
     steps:
     - uses: actions/checkout@v4
     - uses: ./.github/actions/setup-all
+    - run: git fetch origin master:master
```

Comment:
📓 This is what fixes the issue with credo not being able to find the master branch do run a diff.

Reply:
When we run `credo` locally does it find master successfully?

Reply:
Yep.